src/components/Header.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import { Box, Header, Button, Image } from 'grommet';
+import Logo from '../../static/assets/logo.svg';
+
+const linkItems = [
+  { id: '1', label: 'Work', link: '/work' },
+  { id: '2', label: 'Our team', link: '/our-team' },
+  { id: '3', label: 'Blog', link: '/blog' },
+];
+
+export const SiteHeader = () => (
+  <Header pad="medium">
+    <Box
+      width="100wh"
+      justify="stretch"
+      direction="row"
+      gap="medium"
+      margin="medium"
+      align="center"
+    >
+      <Link to="/">
+        <Image src={Logo} alt="Soft Bee" alignSelf="start" href="/" />
+      </Link>
+    </Box>
+    <Box alignSelf="end" direction="row">
```

Comment:
I guess all the elements are not vertically centered:
![image](https://user-images.githubusercontent.com/12502321/84644124-660d4900-af07-11ea-9abd-c5806d28d74d.png)

Consider this change: 
```suggestion
    <Box alignSelf="end" align="center" direction="row">
```